src/Widgets/DataSearchCountWidget/DataSearchCountWidgetComponent.tsx

```diff
@@ -0,0 +1,42 @@
+import { connect, ContentTag, provideComponent, useData } from 'scrivito'
+import {
+  DataSearchCountWidget,
+  DataSearchCountWidgetInstance,
+} from './DataSearchCountWidgetClass'
+import { EditorNote } from '../../Components/EditorNote'
+
+provideComponent(DataSearchCountWidget, ({ widget }) => (
+  <h1 className="h3 text-center">
```

Comment:
* [ ] Why is this an H1? Looks like a potential SEO pitfall. `<h3 className="text-center">`?
* [ ] FYI In general it's sad that we don't enable editors to show the result in any widget with any style they want. E.g., always-centered will not work for sites that use left-aligned headings. It would be great to have, e.g., a conditional widget that can operate on the result count, show different widgets depending on the value, and use a placeholder to show the value. Maybe a CountHeadlineWidget would be a sensible compromise.

Reply:
I added the options `style`, `level` and `alignment`. This hopefully helps to use it in more situations.